HEC.FDA.ViewModel/Hydraulics/GriddedData/GriddedImporterVM.cs

```diff
@@ -203,26 +223,30 @@ private void SaveNew()
         {
             string destinationDirectory = Connection.Instance.HydraulicsDirectory + "\\" + Name;
             Directory.CreateDirectory(destinationDirectory);
-            List<HydraulicProfile> pathProbs = new();
+            List<HydraulicProfile> hydraulicProfiles = new();
             foreach (WaterSurfaceElevationRowItemVM row in ListOfRows)
             {
-                string fileNameFromChildElementDir = getFilePathFromChildElement(row);
-                pathProbs.Add(
-                    new HydraulicProfile(row.Probability, fileNameFromChildElementDir, null));
-                StudyFilesManager.CopyDirectory(row.Path, row.Name, destinationDirectory);
+                string fileNameFromChildElementDir = GetFilePathFromChildElementRoot(row);
```

Comment:
This change was for consistency largely. We consider row.Path in every other importer to be the absolute path to the file that contains the data. Prior to this refactor, in gridded data it actually reffered to the directory, not the file. Because of the refactor I now pull out the source directory from row.path before passing it into the StudyFilesManager.Copy method.